properties.schema

```diff
@@ -7,31 +7,31 @@
     "pageLevelProgress": {
       "type": "string",
       "required": true,
-      "default": "Page sections",
+      "default": "Page regions",
       "inputType": "Text",
       "validators": [],
       "translatable": true
     },
     "pageLevelProgressIndicatorBar": {
       "type": "string",
       "required": true,
-      "default": "Progress bar. Select here to view your current progress, and select an item to navigate to it. You have completed ",
+      "default": "Page progress. Use this to listen to the list of regions in this topic and whether they're completed. You can jump directly to any that are incomplete or which sound particularly interesting.",
       "inputType": "Text",
       "validators": [],
       "translatable": true
     },
     "pageLevelProgressMenuBar": {
       "type": "string",
       "required": true,
-      "default": "You have completed ",
+      "default": "Page completed ",
```

Comment:
`Page completion is `?
`Page completion at `?
@moloko?

Reply:
what's going on here @tomgreenfield ? #107